src/smartpeak/source/core/RawDataProcessor.cpp

```diff
@@ -1129,24 +1163,29 @@ namespace SmartPeak
     LOGD << "END ExtractChromatogramWindows";
   }
 
-  void ExtractSpectraWindows::process(RawDataHandler& rawDataHandler_IO, const std::map<std::string, std::vector<std::map<std::string, std::string>>>& params_I, const Filenames& filenames) const
+  ParameterSet ExtractSpectraWindows::getParameterSchema() const
+  {
+    return ParameterSet();
```

Comment:
We could define these and any other "empty" `getParameterSchema` similar to what you had done for `ParameterSet ProcessSequence::getParameterSchema()` for methods where there are no OpenMS parameters.  Please let me know if there are enough "hints" in the Processor class and parameters.csv file to internally define the schema.

I am thinking we could add incorporate this in this PR.  What are your thoughts?

Reply:
Ah! yes... yes, schema for non OpenMS should do same as ProcessSequence::getParameterSchema() like you say. I missed this one. Is there a lot? I will make a round.

For empty parameter we can have a more explicit static ParameterSet::empty_set() or use same pattern indeed.

Reply:
I Have written the parameter schema for 
MergeInjections
mzML
FIAMS
based on the code and the csv files.

However; 2 Parameters appears in the csv files, but I don't find a reference (the use of that) in the code:
FeaturePlotter
SequenceSegmentPlotter

Let me know if I am missing something.

Reply:
FeaturePlotter and SequenceSegmentPlotter are legacy.  Please ignore and remove.